CHANGELOG.asciidoc

```diff
@@ -46,6 +46,7 @@ endif::[]
 [float]
 ===== Bug fixes
 * Preferring controller names for Spring MVC transactions, `use_path_as_transaction_name` only as a fallback - {pull}2320[#2320]
+* Fixing log4j configuration error - {pull}2343[#2343]
```

Comment:
This needs to be in the 1.28.2 release notes

Reply:
Ohh, of course 🤦 
Thanks